code/game/turfs/closed/walls.dm

```diff
@@ -27,6 +27,8 @@
 	var/sheet_type = /obj/item/stack/sheet/iron
 	var/sheet_amount = 2
 	var/girder_type = /obj/structure/girder
+	/// A turf that will replace this turf when this turf is destroyed
+	var/decon_type
```

Comment:
Do we need both a girder and a decon type? Could these two be combined?

Reply:
id have to do some weird isturf() checks probably, i can do it if you insist though